ocp_resources/network_attachment_definition.py

```diff
@@ -201,31 +201,36 @@ class OVNOverlayNetworkAttachmentDefinition(NetworkAttachmentDefinition):
     def __init__(
         self,
         network_name=None,
+        topology=None,
         **kwargs,
     ):
         """
         Create and manage an OVN k8s overlay NetworkAttachmentDefinition (a switched, layer 2, topology network).
 
         API reference:
-        https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-network-plugin-json-object_configuring-additional-network
+        https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network
 
         Args:
             network_name (str): The name of the network. This field is mandatory when not using yaml
                 file.
+            topology (str): The secondary network topology to be created - "layer2" or "localnet".
```

Comment:
`"layer2" or "localnet".` please remove, we do not want to update this once we have another type.

Reply:
Ack